src/Compilers/CSharp/Portable/Binder/Semantics/OverloadResolution/OverloadResolution.cs

```diff
@@ -1748,11 +1736,13 @@ private BetterResult BetterConversionFromExpression(BoundExpression node, TypeSy
         {
             okToDowngradeToNeither = false;
 
-            if (considerRefKinds)
+            var argumentKind = node.Kind;
+            if (considerRefKinds || argumentKind == BoundKind.OutVariablePendingInference || argumentKind == BoundKind.OutDeconstructVarPendingInference)
```

Comment:
Please revert change to this file then.

---
In reply to: [80800555](https://github.com/dotnet/roslyn/pull/13999#discussion_r80800555) [](ancestors = 80800555,80795408,80794118,80793171,80791006)